spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/config/DataFlowServerConfiguration.java

```diff
@@ -36,13 +36,13 @@
 import org.springframework.boot.autoconfigure.security.oauth2.OAuth2AutoConfiguration;
 import org.springframework.boot.autoconfigure.web.HttpMessageConverters;
 import org.springframework.boot.context.properties.EnableConfigurationProperties;
-import org.springframework.cloud.dataflow.admin.completion.TapOnDestinationRecoveryStrategy;
-import org.springframework.cloud.dataflow.admin.controller.StreamDefinitionController;
-import org.springframework.cloud.dataflow.admin.repository.InMemoryStreamDefinitionRepository;
-import org.springframework.cloud.dataflow.admin.repository.InMemoryTaskDefinitionRepository;
-import org.springframework.cloud.dataflow.admin.repository.StreamDefinitionRepository;
-import org.springframework.cloud.dataflow.admin.repository.TaskDatabaseInitializer;
-import org.springframework.cloud.dataflow.admin.repository.TaskDefinitionRepository;
+import org.springframework.cloud.dataflow.server.completion.TapOnDestinationRecoveryStrategy;
```

Comment:
these are no longer in proper alphabetical order after the name change... should re-org imports not just replace the strings